### ISSUE TITLE
How To Train MTCNN

### ISSUE THREAD

**Issue by herleeyandi** [davidsandberg/facenet#214]
Hello I am new in Tensorflow and deep learning. Now I am in face alignment task. I want to train MTCNN using my own dataset. So anybody here can give me some hint to do that?, for example:
1. Format of the dataset, like naming the folder and image.
2. Train the network to produce the model
3. Test one face image and output the result.
4. Is there ane pretrained model?, if exist, how I used it?

Maybe some of you will think this basic ones in tensorflow, but really I don't get any idea to train this MTCNN, so any other hint and suggestion will be appreciate. Thank you, sorry my English is bad.

**Comment by yekeping** [comment]
@herleeyandi 
it may be not right,but I just change the folder of the author,the accuracy is also my problem.why do I follow the code of the author,but the accuracy is low on my dataset

**Comment by herleeyandi** [comment]
I see, now it make sense. OK I am agree 100% with you. So how can I train with your MTCNN Code?, hint, suggestion, and step?

**Comment by sun9700** [comment]
Will @davidsandberg  release the code of training facenet (with Triplet Loss)  and MTCNN on CASIA-WebFace  and  MS-Celeb-1M ?

**Comment by davidsandberg** [comment]
No such plans right now.
